### DISCUSSION TITLE
issue discussion: email replies should supportmarkdown

### DISCUSSION THREAD

**Discussion by cameron-simpson** [community/community#44937] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I'm an email first person. A text/plain email first person. It was very disheartening to discover that my email reply's markdown wasn't honoured in the issue discussion. There isn't even a toggle in the "Edit" mode for me to retrospectively fix it.

As a result my recent reply wasn't as readable as hoped, and the follow on (from a Windows only not-good-at-iterspersed-replies colleague) was practically illegible.

I think markdown is great, and it would be really nice if markdown in email, _particularly_ text/plain email with no HTML part, were honoured. If you're really concerned about misformatting an email reply I could even arrange to have my github replies be text/markdown for github.

**Comment by jge162** [comment]
@cameron-simpson you can try sending the email as text/markdown or including a markdown version in the HTML part of the email to ensure that the formatting is preserved for those who use email clients that support it. Additionally, you can try using other tools, such as Github, to format your email replies in markdown.

**Comment by ferdinandyb** [comment]
Does sending as `text/markdown` work?
